cmd/query/app/querysvc/query_service.go

```diff
@@ -104,7 +107,16 @@ func (qs QueryService) FindTraces(ctx context.Context, query *spanstore.TraceQue
 	if err != nil {
 		return nil, err
 	}
-	return spanReader.FindTraces(ctx, query)
+	traces, err := spanReader.FindTraces(ctx, query)
+	if !query.RawTraces {
+		for _, trace := range traces {
+			_, err := qs.adjust(trace)
```

Comment:
we need to respect the existing API of the adjusters (or change it) - v1 adjusters expected the caller to use the returned value, not expect a change in place.